From 63755cea896a0fd959595b1472b7bc42784038dc Mon Sep 17 00:00:00 2001
From: H. Peter Anvin <hpa@zytor.com>
Date: Thu, 5 Jun 2008 13:37:45 -0700
Subject: [PATCH] klibc: avoid .gitignore in the include directory

If we put a .gitignore in the include directory, it gets installed by
"make install".  It's easy enough to simply put such excludes in
usr/.gitignore instead.

Signed-off-by: H. Peter Anvin <hpa@zytor.com>
---
 usr/.gitignore               |    2 ++
 usr/include/klibc/.gitignore |    2 --
 2 files changed, 2 insertions(+), 2 deletions(-)
 create mode 100644 usr/.gitignore
 delete mode 100644 usr/include/klibc/.gitignore

diff --git a/usr/.gitignore b/usr/.gitignore
new file mode 100644
index 0000000..77534ee
--- /dev/null
+++ b/usr/.gitignore
@@ -0,0 +1,2 @@
+# Generated headers (we don't want a .gitignore in the include directory!)
+/include/klibc/havesyscall.h
diff --git a/usr/include/klibc/.gitignore b/usr/include/klibc/.gitignore
deleted file mode 100644
index da63bd6..0000000
--- a/usr/include/klibc/.gitignore
+++ /dev/null
@@ -1,2 +0,0 @@
-# generated header
-/havesyscall.h
-- 
1.5.5.1

